src/clients/memberclient.cpp

```diff
@@ -326,6 +367,22 @@ int main(int argc, char** argv)
       add_new(*tls_connection, user_cert_file, add_user_proposal);
     }
 
+    if (*add_node)
+    {
+      const auto new_node_raw_cert = slurp(new_node_cert_file);
+      node_info.cert = new_node_raw_cert;
+      const auto new_node_raw_quote = slurp(new_node_quote_file);
+      node_info.quote = new_node_raw_quote;
+      auto new_node_id = submit_add_node(*tls_connection, node_info);
+      if (new_node_id != INVALID_NODE_ID)
+      {
+        // nodes are untrusted until they are accepted, so a member should
+        // stage a vote to accept the new node (the current member will do just
+        // fine).
+        submit_accept_node(*tls_connection, new_node_id);
```

Comment:
I am not sure this is needed here. Could we print the `NodeId` of the node which has just been added and then let members submit their votes to accept that node instead - just to make sure that the logic of `memberclient.cpp` stays simple enough?